packages/connectivity/connectivity_macos/example/pubspec.yaml

```diff
@@ -0,0 +1,18 @@
+name: connectivity_example
+description: Demonstrates how to use the connectivity plugin.
+
+dependencies:
+  flutter:
+    sdk: flutter
+  connectivity: any
+  connectivity_macos:
+    path: ../
```

Comment:
I think this should depend in the correct version of `connectivity` that federates connectivity_macos, that way people can work off of this example with the correct dependency on the `connectivity` plugin.